src/main/java/edu/uc/cech/agrawadv/eventorganizer/resource/ResourceService.java

```diff
@@ -5,6 +5,7 @@
 import java.util.UUID;
 
 public class ResourceService {
+	
```

Comment:
Added spacing because the rest of your code has a one line above and below the code for readability.  ref :https://css-tricks.com/write-better-code-3-levels-code-consistency/